src/index.html

```diff
@@ -6,8 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="normalize.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```

checklist (read point 1, but you can do not leave empty lines if the tag contains only 1 line)